libs/design/src/molecules/callout/callout.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="daff-callout__container">
+  <ng-content></ng-content>
```

Comment:
This should probably define some order of title/body.

Reply:
Resolved in e7ff18d